### PR TITLE
Proof GIMBAL_LOCK_AVOIDANCE #585

### PR DESCRIPTION
**Pull request by rcornbtree**
Full file checked:

- Line 63: Unsure if period or commas -- left unchanged from scan

### CODE REVIEW

File: Luminary099/GIMBAL_LOCK_AVOIDANCE.agc [hunk 1]
```diff
@@ -56,7 +56,7 @@ WCALC		LXC,1	DLOAD*
 			5
 		STORE	TM		# MANEUVER EXECUTION TIME SCALED AS T2
 		SETGO
-			CALCMAN2	# D(OFF) = CONTINUE MANEUVER
+			CALCMAN2	# 0(OFF) = CONTINUE MANEUVER
 			NEWANGL +1	# 1(ON) = START MANEUVER
 
 # THE FOUR SELECTABLE FREE FALL MANEUVER RATES SELECTED BY
```
Comment (wopian) [inline Luminary099/GIMBAL_LOCK_AVOIDANCE.agc:62]: Line below this doesn't have spaces between the numbers.

R0030 https://www.ibiblio.org/apollo/ScansForConversion/Luminary099/0364.jpg

```diff
- # LOADING RATEINDX WITH 0, 2, 4, 6, RESPECTIVELY
+ # LOADING RATEINDX WITH 0,2,4,6, RESPECTIVELY
```

**Review by wopian — CHANGES_REQUESTED** [review]
PR additions themselves are good, just one missed transcription error.

**Comment by rcornbtree** [comment]
Good catch, removed those spaces -- thanks!

**Review by wopian — APPROVED** [review]

**Comment by wopian** [comment]
Thank you for contributing 🙂

**Pull request merged**